src/components/Profile/Profile.js

```diff
@@ -63,7 +151,8 @@ class UserProfile extends Component {
     if (isAuthenticated) {
       return (
         <Fragment>
-          <Form layout="vertical">
+          <Form layout="vertical" onSubmit={this.handleSubmit}>
+          {msg ? <Alert message={msg} type="error" /> : null}
```

Comment:
This message shall be the one received from the server. and for type check the status if 200 type=success otherwise type=error